wiki/about/modders.md

```diff
@@ -91,6 +91,14 @@ Ended up maintaining a wild variety of mods after adopting them from their origi
 
 [GitHub](https://github.com/erisapps) | [Donate with Patreon](https://www.patreon.com/erisapps) | [Donate with Ko-fi](https://ko-fi.com/erisapps)
 
+## Exomanz
+Note: don't forget to add their description.
```

Comment:
Would rather use the older revision or a different variation that does not imply the description is incomplete by mistake

Reply:
Pushed a new commit to my fork which has an updated bio. I can see how the previous version could've been confusing